app/src/main/java/com/onboarding/parkingsystemkotlin/activity/ShowReservationsActivity.kt

```diff
@@ -28,7 +25,7 @@ class ShowReservationsActivity : AppCompatActivity() {
     }
 
     private fun setListeners() {
-        binding.btnShowReservationsReturn.setOnClickListener { presenter.onReturnButtonPressed() }
+        binding.buttonShowReservationsReturn.setOnClickListener { onBackPressed() }
     }
```

Comment:
you are setting one listener, so rename in singular pls

Reply:
name changed

Reply:
renamed